jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/MetaInfConfiguration.java

```diff
@@ -696,34 +695,23 @@ protected List<Resource> findWebInfLibJars(WebAppContext context)
         if (webInf == null || !webInf.exists())
             return null;
 
-        List<Resource> jarResources = new ArrayList<Resource>();
+        List<Resource> jarResources = new ArrayList<>();
         Resource webInfLib = webInf.resolve("/lib");
-        if (webInfLib.exists() && webInfLib.isDirectory())
+        for (Resource lib: webInfLib) // walk all "lib" hits
```

Comment:
Not needed, see ee10 comment for the similar code.

Reply:
Done.